Review Getting Started with Codewind in Kabanero Guide

j-c-berger opened the pull request:
Making this PR for review of the `Getting started with Codewind in Kabanero` guide. 

Recent changes to the guide include:
- link to `Working with Kabanero Collections` guide under this guide's `Kabanero Overview` section 
- link to `Appsody Welcome Page` under this guide's `Creating an Appsody project` section 
-  minimum versions of Docker and Eclipse

This guide is in response to `eclipse/codewind` issue #912

j-c-berger committed:
Update README.adoc

nateziemann commented:
I created a PR with some edits.  What is missing is how to configure Codewind to use the stacks from a CollectionHub vs the public AppsodyHub.  This guide needs to be written in the context of a Kabanero workflow, not a generic Codewind workflow.

j-c-berger committed:
Update README.adoc

j-c-berger commented:
@nateziemann @ralanlittle 

I updated this PR reflecting all the necessary changes made to the guide. I've implemented all of Nate's changes and both of your feedback. 

Both under `Developing with VS Code` and `Developing with Eclipse`, I've added the section `Configuring Codewind with Kabanero Collections` to include the steps to configure Codewind to use Kabanero Collections. 

Thank you both for your continued effort and support on this guide--JB

ralanlittle reviewed (approved):
I went through it, it looks good
